docs/source/platform/schema-validation.md

```diff
@@ -1,60 +1,101 @@
 ---
 title: Validating schema changes
-description: How to validate your schema in your existing CI workflow
+description: How to maintain the schema's contract with CI
 ---
 
-The Apollo GraphQL Platform allows developers to confidently iterate a GraphQL schema by validating the new schema against field-level usage data from the previous schema. By knowing exactly which clients will be broken by a new schema, developers can avoid inadvertently deploying a breaking change.
+To recognize the need for schema validation, first understand that a GraphQL schema defines a contract between clients and server that contains the available types and their behavior. When a GraphQL API is deployed, consumers start to request fields and depend on the contract. If the schema is updated, such as adding a field or removing a type, the contract changes. Modifying the schema and contract can have a wide range of impact on clients from positive, more functionality, to adverse, active schema dependencies no longer exist.
 
-A GraphQL schema can change in a number of ways between releases and, depending on the type of change, can affect clients in a variety of ways. Since changes can range from "decidedly safe" to "certain breakage", it's helpful to use schema tools which are aware of actual API usage.
+The Apollo Platform ensures teams deploy schemas without breaking consumers. To prevent dangerous schema evolution, the `apollo service:check` command compares a proposed schema against the active schema to create a list of changes. This resulting list is matched against the field-level usage data from the previous schema and categorizes the changes by severity. If any change's severity is too high, the team is promptly flagged by the cli or GitHub status with actionable feedback.
 
-By comparing a new schema to the last published schema, the Apollo Platform can highlight points of concern by showing detailed schema changes alongside current usage information for those fields. With this pairing of data, the risks of changes can be greatly reduced.
+<h2 id="cli">Setup `apollo` for schema changes</h2>
 
+To check and validate the difference between the current published schema and a new version, run the `apollo service:check` command during continuous integration.
 
-<h2 id="versioning">Understanding schema changes</h2>
+For basic usage, use the following command, substituting the appropriate GraphQL endpoint URL and an API key obtained from the service _Settings_ menu in the [Apollo UI](https://engine.apollographql.com/):
 
-Versioning is a technique to prevent necessary changes from becoming "breaking changes" which affect the existing consumers of an API.  These iterations might be as trivial as renaming a field, or as substantial as refactoring the whole data model.
+```bash
+npx apollo service:check --key="<API_KEY>" --endpoint="http://localhost:4000/graphql"
```

Comment:
Don't we expect users to globally install the `apollo` package?

Reply:
nit: Maybe `--endpoint="http://localhost:1234/REPLACE_ME"` to make it more obvious? Unless we're trying to make it copy-able for Apollo Server users, for which I assume this is the default configuration? If so, ignore me 😉